test/e2e/specs/fakerp/clusterstatus.go

```diff
@@ -0,0 +1,30 @@
+package fakerp
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/openshift-azure/test/clients/azure"
+)
+
+var _ = Describe("Cluster Status E2E tests [ClusterStatus][Fake][EveryPR]", func() {
+	var (
+		cli *azure.Client
+	)
+
+	BeforeEach(func() {
+		var err error
+		cli, err = azure.NewClientFromEnvironment(false)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("should allow an SRE to fetch the status of control plane pods", func() {
+		By("Using the OSA admin client")
+		status, err := cli.OpenShiftManagedClustersAdmin.ClusterStatus(context.Background(), os.Getenv("RESOURCEGROUP"), os.Getenv("RESOURCEGROUP"))
+		Expect(err).NotTo(HaveOccurred())
+		Expect(status).NotTo(BeNil())
```

Comment:
go on, unmarshal status into an interface{} and use jsonpath to check that it contains an expected pod.

Reply:
Shouldn't you be able to unmarshal directly to `[]v1.Pod` and then ensure that the list contains the actual number of pods you expect (3 etcd servers, 3 api servers, 3 controller managers, 3 schedulers) ?

Reply:
Actually you should be able to unmarshal to `adminclient.OpenShiftManagedClustersStatus`.

Reply:
Actually no.2, the client already returns a `adminclient.OpenShiftManagedClustersStatus` so you can directly iterate over the returned pods.

Reply:
> Shouldn't you be able to unmarshal directly to []v1.Pod and then ensure that the list contains the actual number of pods you expect (3 etcd servers, 3 api servers, 3 controller managers, 3 schedulers) ?

unmarshalling directly to []v1.Pod is also fine.  I don't want to test the length of the list of pods - ensuring that a particular one is there (e.g. the apiserver for master-000000) is good enough for me.